client/components/backup-storage-space/hooks.tsx

```diff
@@ -62,16 +43,13 @@ export const useStorageUsageText = (
 
 		if ( availableUnit === StorageUnits.Gigabyte ) {
 			return translate(
-				'%(usedGigabytes).1fGB used of %(availableUnitAmount)dGB - {{a}}Upgrade{{/a}}',
-				'%(usedGigabytes).1fGB used of %(availableUnitAmount)dGB - {{a}}Upgrade{{/a}}',
+				'%(usedGigabytes).1fGB used of %(availableUnitAmount)dGB',
+				'%(usedGigabytes).1fGB used of %(availableUnitAmount)dGB',
 				{
 					count: usedGigabytes,
 					args: { usedGigabytes, availableUnitAmount },
 					comment:
 						'Must use unit abbreviation; describes used vs available storage amounts (e.g., 20.0GB of 30GB used, 0.5GB of 20GB used)',
-					components: {
-						a: <a onClick={ onClick } href={ storageUpgradeUrl } />,
-					},
```

Comment:
I'm noticing this hook, `useStorageUsageText(bytesUsed, bytesAvailable)` has some unit-tests associated with it that are now failing. They will need to be updated in `client/components/backup-storage-space/test/hooks.js`. 😉

Reply:
Thanks, @elliottprogrammer, I too saw that those tests were failing. I thought I will post the comment about it when I created this PR and get it fixed after everyone reviews this PR and it is good enough to merge

Reply:
Ok, sounds good. 👍